enterprise/config/config.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"go.uber.org/zap"
+)
+
+type Conf struct {
+	PubKey          string
```

Comment:
comment on public functions/variables

Also we should use more descriptive names if not too long, e.g. Conf -> Config, Iss -> Issuer? .

// Conf is ....

// PubKey is ...